editor/icons/SetDefaultChildParent.svg

```diff
@@ -0,0 +1 @@
+<svg height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="M4 5a2 2 0 1 0 -2 0v1.5a2 2 0 1 0 2.75 2.75h1.5A2 2 0 0 0 7 10v1.5a2 2 0 1 0 2.75 2.75h1.5v-2h-1.5A2 2 0 0 0 9 11.5V10a2 2 0 1 0-2.75-2.75h-1.5A2 2 0 0 0 4 6.5V5z" fill="#e0e0e0"/><path d="M11.25 14.25a2 2 0 1 0 0 -2z" fill="#70bafa"/><path d="M 6 2.5a8 6 0 0 1 8 6h1l-2 2l-2 -2h1a6 4 0 0 0 -6 -4z" fill="#5fff97"></svg>
```

Comment:
```suggestion
<svg height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="M4 5a2 2 0 1 0 -2 0v1.5a2 2 0 1 0 2.75 2.75h1.5A2 2 0 0 0 7 10v1.5a2 2 0 1 0 2.75 2.75h1.5v-2h-1.5A2 2 0 0 0 9 11.5V10a2 2 0 1 0-2.75-2.75h-1.5A2 2 0 0 0 4 6.5V5z" fill="#e0e0e0"/><path d="M11.25 14.25a2 2 0 1 0 0 -2z" fill="#70bafa"/><path d="M 6 2.5a8 6 0 0 1 8 6h1l-2 2l-2 -2h1a6 4 0 0 0 -6 -4z" fill="#5fff97"/></svg>
```
You have to close the `path` element

Reply:
ahaha whoops, this is what happens when I handcraft the XML instead of installing an editor, tyvm 😅